src/Symfony/Component/Serializer/Encoder/XmlEncoder.php

```diff
@@ -30,15 +30,18 @@ class XmlEncoder extends SerializerAwareEncoder implements EncoderInterface, Dec
     private $format;
     private $context;
     private $rootNodeName = 'response';
+    private $loadOptions;
 
     /**
      * Construct new XmlEncoder and allow to change the root node element name.
      *
-     * @param string $rootNodeName
+     * @param string   $rootNodeName
+     * @param int|null $loadOptions
```

Comment:
We should add a short explanation what `$loadOptions` means (something like "a bit field of LIBXML_ constants").